web-marketplace/src/pages/Dashboard/hooks/useProfileItems.tsx

```diff
@@ -4,20 +4,20 @@ import { useQueryIsIssuer } from 'hooks/useQueryIsIssuer';
 import { useQueryIsProjectAdmin } from 'hooks/useQueryIsProjectAdmin';
 
 export const useProfileItems = () => {
-  const isIssuer = useQueryIsIssuer();
-  const isCreditClassCreator = useQueryIfCreditClassCreator();
-  const isProjectAdmin = useQueryIsProjectAdmin();
+  const isIssuer = useQueryIsIssuer({});
```

Comment:
can we just use the following to avoid the weird `isIssuer.isIssuer` below

```suggestion
  const { isIssuer } = useQueryIsIssuer({});
```
ditto for `isProjectAdmin.isProjectAdmin`

Reply:
This is a leftover from the rebase, will fix it.

Reply:
> ditto for `isProjectAdmin.isProjectAdmin`

Reply:
@flagrede

Reply:
Oh missed that one.

Reply:
fixed btw.